bats/config.bats

```diff
@@ -164,3 +164,22 @@ teardown() {
     regex='John Doe <john@doe.com>'
     [[ "$output" =~ "$regex" ]] || false
 }
+
+@test "sql-server: starting test without server throws error" {
+    dolt config --global --add user.name "bats tester"
+    dolt config --global --add user.email "joshn@doe.com"
+
+    dolt init
+    dolt sql -q "
+    CREATE TABLE test (
+       pk int primary key
+    )"
+
+    dolt config --global --unset user.name
+    dolt config --global --unset user.email
+
+    run dolt sql -q "SET @@dolt_repo_$$_head = COMMIT('-a', '-m', 'updated stuff')"
+    echo $output
+    [ "$status" -eq 1 ]
+    [[ "$output" =~ "Aborting commit due to empty committer name. Is your config set" ]] || false
```

Comment:
I think you should also add a test case in this test if `user.name` is set but `user.email` is not.